.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Onboard a new team member
+about: Used for new hires within the U.S. Tax Court and vendor teams to grant access to all relevant accounts and tools.
+title: Onboard [Person]
+---
+
+Welcome to the team! 🎉 😎
+
+#### Access to tools
+
+- [ ] GitHub — speak with your team lead to determine the appropriate repositories and levels of access in the [ustaxcourt](https://github.com/ustaxcourt) and [flexion](https://github.com/flexion) organizations.
+- [ ] [Slack](https://ustaxcourt.slack.com) — look for important channels to join in [How We Work](./how-we-work.md).
+- [ ] [Sonarcloud](https://sonarcloud.io/organizations/flexion-github/projects)
```

Comment:
The team's reliance on Sonarcloud has been replaced with Deepscan.

Reply:
Exciting! I see [Sonarcloud is still referenced quite a bit](https://github.com/ustaxcourt/ef-cms/search?q=Sonarcloud) — has it been fully deprecated and removed, or are we still in the process of replacing it? 

I’m happy to remove all references to Sonarcloud, if we’re not using it anymore.

Reply:
Amendment: the removal of Sonarcloud will be completed after Batch 1 is merged.